### PR TITLE
zh:news/releases/1.12.x/announcing-1.12.1/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.12.x/announcing-1.12.1/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Istio 1.12.1 发布公告
+linktitle: 1.12.1
+subtitle: 补丁发布
+description: Istio 1.12.1 补丁发布
```
Comment (ericvn) [inline content/zh/news/releases/1.12.x/announcing-1.12.1/index.md:5]:
```suggestion
description: Istio 1.12.1 补丁发布.
```

**Review by ericvn — COMMENTED** [review]
To fix lint error

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**